src/sdk/sdk.ts

```diff
@@ -110,6 +110,7 @@ import {
   ExchangeService,
   CrossChainBridgeBuildTXResponse,
   CrossChainQuote,
+  MultiChainQuotes,
```

Comment:
What is the difference between CrossChainQuotes and MultiChainQuotes? Why do we need two separate data structures to achieve the same thing?

Can't we reuse the same structure?

Reply:
CrossChainQuote could include more fields from Lifi which cannot be shared with socketv2, socketv2 has its own data structure too